Cows-HacknSlash/Assets/Scripts/Spells/FireTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FireTest : MonoBehaviour {
+
+    [SerializeField]
+    [Range (1, 45)]
+    private float FiringAngle = 45f;
+
+    [SerializeField]
+    private float Gravity = 9.8f;
+
+    [SerializeField]
+    private float WaitTime = 0f;
+
+    private Vector3 target;
+	
+    public void Shoot(Transform FirePrefab) {
+         StartCoroutine(FireMovement(FirePrefab));
+    }
+
+    private IEnumerator FireMovement(Transform FirePrefab) {
+        yield return new WaitForSeconds(WaitTime);
+        FirePrefab.position = transform.position;
+
+        // getting the mouse input position
```

Comment:
Capitalize the G of getting and a full period (same for all other comments in this file)